skfda/exploratory/stats/_stats.py

```diff
@@ -91,3 +91,38 @@ def depth_based_median(fdatagrid, depth_method=modified_band_depth):
 
     # The median is the deepest curve
     return fdatagrid[indices_descending_depth[0]]
+
+
+def trimmed_means(fdatagrid,
```

Comment:
Maybe should we use same name of function and parameters as numpy (https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.trim_mean.html)?

Reply:
done